cyber/scheduler/policy/choreography_context.cc

```diff
@@ -17,6 +17,7 @@
 #include "cyber/scheduler/policy/choreography_context.h"
 
 #include <climits>
```

Comment:
The `<climits>` header can be safely deleted now as XXX_MAX/MIN was no longer used ?